src/context.py

```diff
@@ -3,11 +3,50 @@
 from config import MAILBOX
 from commonbot.utils import combine_message, send_message
 
+class ReportModal(discord.ui.Modal):
+    def __init__(self, *, message: discord.Message):
+        super().__init__(title="Report a message")
+
+        self.message: discord.Message = message
+        self.comments_input: discord.ui.TextInput = discord.ui.TextInput(
+            label="Comments",
+            placeholder="(Optional) Additional comments or context",
+            style=discord.TextStyle.long,
+            required=False,
+            max_length=250
+        )
+        self.add_item(self.comments_input)
+
+    async def on_submit(self, interaction: discord.Interaction):
+        embed: discord.Embed = discord.Embed(
+            title=f"\N{BALLOT BOX WITH BALLOT} Report from #{self.message.channel.name}",
```

Comment:
> \N{BALLOT BOX WITH BALLOT}

Is this a thing?

Reply:
Yep! You can find most emoji by searching their display names in the fountain of knowledge:  
http://www.unicode.org/Public/UCD/latest/ucd/NamesList.txt

Mini-jukebox keeps all its emojis in the strings file:  
https://github.com/b-b-blueberry/mini-jukebox/blob/master/src/strings.py